index.js

```diff
@@ -377,6 +377,22 @@ module.exports = {
     },
 
     /**
+     * Call this to enable forked type checking for TypeScript loader
+     * https://github.com/TypeStrong/ts-loader/blob/v2.3.0/README.md#faster-builds
+     *
+     * This is a build optimization API to reduce build times.
+     *
+     *
```

Comment:
We usually only have one empty (well `*`) line between the description and the start of the arguments. So, delete line 384 :).